### PR TITLE
Add operator sig meeting date

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

**Review by arminru — APPROVED** [review]

**Review by jaronoff97 — APPROVED** [review]
Thank you :bow:

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.g
 eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1pAxBekk6BadHpK717TbhYtBMOlyrgWSvp3QeYDPwwvk/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/1mHNnlCmO0XKUu0xPnKko3GOh0mxVoxjRiN8xmHRImew/edit#)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
 Client Instrumentation|Every Wednesday at 8:00 AM PT|[Google Doc](https://docs.google.com/document/d/16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w/edit)|[Slack](https://cloud-native.slack.com/archives/C0239SYARD2)|
-Kubernetes Operator|N/A|[Google Doc](https://docs.google.com/document/d/1zBNzh1dw9HpXUexTaURRS7n_NQSzqNJSuqNVw9GhIe4/edit#)|[Slack](https://cloud-native.slack.com/archives/C033BJ8BASU)|
+Kubernetes Operator|Every other week on Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1zBNzh1dw9HpXUexTaURRS7n_NQSzqNJSuqNVw9GhIe4/edit#)|[Slack](https://cloud-native.slack.com/archives/C033BJ8BASU)|
```
Comment (trask) [inline README.md:150]: can you update with the new link?

Comment (pavolloffay) [reply README.md]: done

**Review by trask — APPROVED** [review]
(@pavolloffay I updated the doc links in the calendar event)

**Pull request merged**